src/evm/protocol/uniswap_v4/hooks/constants.rs

```diff
@@ -1,2 +1,3 @@
 #![allow(dead_code)]
 pub const POOL_MANAGER_BYTECODE: &[u8] = include_bytes!("pool_manager_bytecode.bin");
+pub const BUNNI_HOOK_BYTECODE: &[u8] = include_bytes!("bunni_hook_bytecode.bin");
```

Comment:
I'm not 100% sure if we will keep this here or it should be moved to be used only during testing but this is to be decided once we know more about the indexer side

Reply:
actually added a new task for it [here](https://propeller-heads.atlassian.net/jira/software/projects/ENG/boards/24?selectedIssue=ENG-4596)